src/Modules/Build.lua

```diff
@@ -1261,13 +1261,16 @@ function buildMode:RefreshSkillSelectControls(controls, mainGroup, suffix)
 	end
 end
 
-function buildMode:FormatStat(statData, statVal, overCapStatVal)
+function buildMode:FormatStat(statData, statVal, overCapStatVal, pool)
 	if type(statVal) == "table" then return "" end
 	local val = statVal * ((statData.pc or statData.mod) and 100 or 1) - (statData.mod and 100 or 0)
 	local color = (statVal >= 0 and "^7" or statData.chaosInoc and "^8" or colorCodes.NEGATIVE)
 	if statData.label == "Unreserved Life" and statVal == 0 then
 		color = colorCodes.NEGATIVE
 	end
+	if statData.label and statData.label:match("Cost$") and statVal and pool and statVal > pool then
```

Comment:
This incorrectly marks EB as a red cost.
![image](https://user-images.githubusercontent.com/31533893/190880436-e94c1fc7-2535-40c4-a6cb-9ea8c2cc79f1.png)

Reply:
Should be fixed by https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/5019/commits/b6705ebfa0964af51ce5958f1735d5812389786b